### PR TITLE
References to the word 'migration'

### PR DESCRIPTION
**Pull request by tompods**

### CODE REVIEW

File: docs/HowTo/centos8-eol-solution.md [hunk 1]
```diff
@@ -17,20 +17,20 @@ It is worth mentioning that both CentOS and RHEL and EuroLinux are systems
...
 
-The operation of switching support is simple, safe and completely reversible.
-What is very important, it requires neither reinstallation of the system nor
-the applications installed on it. The process consists of switching the
-repository, installing the el-release package and updating the system. After
-the switch, CentOS will still be usable, even in production environments. 
+The operation of switching support is simple, reasonably safe and completely
+reversible.  What is very important, it requires neither reinstallation of the
+system nor the applications installed on it. The process consists of switching
+the repository, installing the el-release package and updating the system.
```
Comment (AlexBaranowski) [inline docs/HowTo/centos8-eol-solution.md:23]: And reinstalling the system packages inplace?

File: docs/HowTo/centos8-eol-solution.md [hunk 2]
```diff
@@ -17,20 +17,20 @@ It is worth mentioning that both CentOS and RHEL and EuroLinux are systems
...
+After the switch, CentOS will still be usable, even in production environments.
```
Comment (AlexBaranowski) [inline docs/HowTo/centos8-eol-solution.md:24]: After in-place migration

**Review by AlexBaranowski — CHANGES_REQUESTED** [review]

File: docs/HowTo/centos8-eol-solution.md (hunk 2 above)
Comment (tompods) [reply docs/HowTo/centos8-eol-solution.md]: Changed

File: docs/HowTo/centos8-eol-solution.md (hunk 1 above)
Comment (tompods) [reply docs/HowTo/centos8-eol-solution.md]: Changed

**Review by AlexBaranowski — APPROVED** [review]

**Pull request merged**